gravityflow/gravityflowflowchart
# Fix issue with Flowchart Printing

## Description
The flowchart print does not seem to work for the current Flowchart ver 1.3. It was working fine with the Flowchart ver 1.2.

This PR undoes some of the JS updates which had been added to fix the flowchart icon for display on Gravity Forms 2.5 beta. The CSS updates that were added later suffice for the flowchart icon location with Gravity Forms 2.5 beta, and the JS update, which seems to have broken the printing, can be removed.

## Testing instructions
- Need to confirm Gravity Flow Flowchart icon displays well with both Gravity Forms 2.4 and Gravity Forms 2.5 beta.
- Need to check the printing of the flowchart with both Gravity Forms 2.4 and Gravity Forms 2.5 beta.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
